tests/foreman/virtwho/utils.py

```diff
@@ -139,11 +139,15 @@ def get_virtwho_status():
     """Return the status of virt-who service, it will help us to know
     the virt-who configure file is deployed or not.
     """
+    _, logs = runcmd('cat /var/log/rhsm/rhsm.log')
+    error = len(re.findall(r'\[.*ERROR.*\]', logs))
```

Comment:
If that,  you need to update deploy_validation() as well:
```
    status = get_virtwho_status()
    if status != 'running':
        raise VirtWhoError("Failed to start virt-who service")
................

```

Reply:
we still net 'logs' to get hypervisor_name and guest_name.
So just remove:
`error = len(re.findall(r'\[.*ERROR.*\]', logs))`